### PR TITLE
REL: Bump version numbers for beta 6

### PR DESCRIPTION
**Pull request by ElDeveloper**
And correct channel information in installation instructions.

### CODE REVIEW

File: emperor/__init__.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 #
 # The full license is in the file LICENSE.md, distributed with this software.
 # ----------------------------------------------------------------------------
-__version__ = "1.0.0beta5"  # noqa
+__version__ = "1.0.0beta6"  # noqa
```
Comment (wasade) [inline emperor/__init__.py]: nonblocking, but can't this be pulled from `pkg_resources` like is done in q2?

Comment (ElDeveloper) [inline emperor/__init__.py]: Great point, done!

**Comment by wasade** [comment]
I'm :+1:, do you need a second reviewer or shall I merge?

**Comment by ElDeveloper** [comment]
No second reviewer needed :)

**Comment by ElDeveloper** [comment]
Thanks @wasade!

**Pull request merged**